bindings/pyroot/pythonizations/python/ROOT/_pythonization/_rdataframe.py

```diff
@@ -177,6 +177,28 @@ def pypowarray(numpyvec, pow):
 from ._pyz_utils import MethodTemplateGetter, MethodTemplateWrapper
 
 
+# For refernces to keep alive the NumPy arrays that are read by
+# MakeNumpyDataFrame.
+_numpy_data = {}
+
+
+def _numpy_data_add_ref(data):
+    _numpy_data[id(data)] = data
+
+
+def _numpy_data_remove_ref(key_ll):
+    """Internal function to remove elements from _numpy_data, given a
+    LowLevelView as a key.
+
+    This function is called in C++ from the relevant RDataSource destructor. It
+    needs to be declared at the module level for lifetime reasons, because the
+    C++ side only has a reference to it via std::function.
+    """
+    import ROOT
```

Comment:
Just to try, can we see if `import cppyy` works? In this other PR https://github.com/root-project/root/pull/15011 it was not necessary to go through `ROOT.module` and it would just look better if we could use directly `cppyy` IMHO

Reply:
Ok, I did that